evaluate/repository.go

```diff
@@ -16,7 +16,7 @@ import (
 )
 
 // Repository evaluate a repository with the given model and language.
-func Repository(logger *log.Logger, resultPath string, model evalmodel.Model, language language.Language, testDataPath string, repositoryPath string) (repositoryAssessment metrics.Assessments, problems []error, err error) {
+func Repository(logger *log.Logger, resultPath string, model evalmodel.Model, language language.Language, temporaryRepositoryPath string, repositoryPath string) (repositoryAssessment metrics.Assessments, problems []error, err error) {
```

Comment:
I think the names `temporaryRepositoryPath` and `repositoryPath` might be confusing... How about keeping `testDataPath` and renaming `repositoryPath` to `repositoryName` since it is only used for logging as far as I can see.